WikiWatershed/model-my-watershed
# Show Area of Interest Info In Modeling Stage

## Overview

We show the AoI info in "Analyze" but lose the view in Model. The user may still be interested in the AoI information at the model stage, so this PR....
- Makes the analyze view in Model look identical to the analyze view in Analyze (add aoi header, loses model controls ... keeps project header)
- Adds a "details" button to the project header that displays the aoi name and size, as well as the model name and description. There are "edit" buttons that lead the user to either -- change the AoI (same as new project button) or change the model (routes to analyze page)

[Wireframe](https://user-images.githubusercontent.com/1809908/29676258-92ed355e-88c5-11e7-916a-9393a7cebbb4.png)


Connects #1366 

### Demo

<img width="705" alt="screen shot 2017-09-01 at 1 09 05 pm" src="https://user-images.githubusercontent.com/7633670/29980974-f9615608-8f19-11e7-8c88-98fda20e1cde.png">


<img width="706" alt="screen shot 2017-09-01 at 1 09 14 pm" src="https://user-images.githubusercontent.com/7633670/29980977-fc17ab22-8f19-11e7-82ea-b43f876190bc.png">

![screen shot 2017-09-01 at 1 32 41 pm](https://user-images.githubusercontent.com/7633670/29981002-105306c2-8f1a-11e7-8a16-eb66ccd33201.png)

## Notes
- I couldn't find the icon used in the wireframe (file icon with search icon inside). I chose a similar one (file icon with lines of text)

## Testing Instructions

 *  Pull and `bundle`
 * For each model, toggle between analyze and model views, and open the "details" dropdown
 * Choose an AoI with a long name (eg Petty-Island River HUC-12) and confirm the views still scroll all the way to the bottom and the details view fits all the information
